brainlessbuild.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
+
+git submodule init
+git submodule update
+
+(
+	cd dependencies/physfs
+	cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .
+	make
+	make install
+)
+
+(
+	cd dependencies/glm
```

Comment:
It may be easier to add the local GLM header paths to cmake directly too, as it's just a header-only library it shouldn't need to install anything, just setup the include paths...